extensions/wikia/ArticleVideo/ArticleVideoController.class.php

```diff
@@ -4,9 +4,10 @@ class ArticleVideoController extends WikiaController {
 
 	private function fallbackLanguage( $lang ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this function 4, down to the maximum allowed 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)

[project-id]: wikia:app